components/button/index.js

```diff
@@ -96,7 +96,7 @@ LinkButton.propTypes = {
   text: string.isRequired,
 }
 
-const ApplyLink = ({ href = '/apply', text = 'Apply to join us', ...props } ) => (
+const ApplyLink = ({ href = '/apply', text = 'Join the waitlist', ...props } ) => (
```

Comment:
if I read the issue correctly, the new text should only be used in the footer, not in the home page hero. This could be achieved by passing a custom `text` prop to the `ApplyLink` component in the footer instead of changing the default text value here.

Reply:
The default text is used because it is the same for hero page and footer. I believe Chris did the Hero page, but I mentioned to him that I will take care of the button